app/views/home/_vertical_menu.html.erb

```diff
@@ -16,10 +16,10 @@
           <%= menu_secondary_item item %>
         <% when Menu::Divider %>
           <%= content_tag(:div, "", :class => "divider") %>
-          <%= content_tag(:span, _(item.caption), :class => "nav-item-pf-header") if item.caption %>
+          <%= content_tag(:span, menu_title, :class => "nav-item-pf-header") if item.caption %>
```

Comment:
This gives all the sub-headers the title of the main menu
![screenshot from 2017-12-04 13-47-09](https://user-images.githubusercontent.com/1540531/33551315-abcc977a-d8f9-11e7-98cf-b25753e76d7d.png)